simple-platform/build.gradle.kts

```diff
@@ -22,9 +22,15 @@ android {
       )
     }
   }
+
+  compileOptions {
```

Comment:
Hmm, why do we need to turn this on specifically for this module? Doesn't turning it on in the main `app` module automatically take care of it for submodules as well?

Reply:
Turns out, it doesn't. Not sure why it was working before.
https://developer.android.com/studio/write/java8-support

![image](https://user-images.githubusercontent.com/6140516/127476428-fc2dfae6-0e06-4bc7-8469-185fbbaf746a.png)